Update k8gb curl demo to be usable for real deployments (#506)
k8gb-io/k8gb: CONTRIBUTING.md

- Open the `http://localhost:4000/` page in your browser.
- Website will automatically rebuild and refresh in the browser to accommodate the related code changes.

> ### End-to-end demo helper

Reviewer: @ytsarev suggestion to promote the header level, as "End-to-end demo helper" is not the child of  "k8gb.io website"
```suggestion
## End-to-end demo helper
```

The demo helper is designed to work with `podinfo` that was deployed by

```sh
make deploy-test-apps
```

> It will configure `podinfo` to expose geo tag as part of http response.

Reviewer: @ytsarev just a few grammar fixes:
```suggestion
It will configure `podinfo` to expose geotag as part of an HTTP response.
```

To test and/or demonstrate continuous query to GSLB enabled endpoint execute

```sh
make demo DEMO_URL=https://failover.test.exampledns.tk
```

The happy path will look like:

```sh
[Thu May 27 15:35:26 UTC 2021] ...

200  "message": "eu-west-1",

[Thu May 27 15:35:31 UTC 2021] ...

200
  "message": "eu-west-1",
[Thu May 27 15:35:36 UTC 2021] ...
```

> The sources for demo helper images can be found [here](./deploy/test-apps/curldemo/)

Reviewer 2: ```suggestion
The sources for demo helper images can be found [here](deploy/test-apps/curldemo)
```
